mapie/tests/test_classification.py

```diff
@@ -351,18 +353,6 @@
             agg_scores="mean"
         )
     ),
-    "raps_randomized_split": (
```

Comment:
Why do we delete `raps_randomized_split`?

Reply:
Done 👍